inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -468,4 +470,35 @@ public function update_safelist_items( $old_version ) {
 
 		update_option( 'wp_rocket_settings', $options );
 	}
+
+	/**
+	 * Display a notification on wrong license.
+	 *
+	 * @return void
+	 */
+	public function display_wrong_license_notice() {
+		if ( ! $this->can_display_notice( false ) ) {
+			return;
+		}
+
+		$main_message = __( "We couldn't generate the used CSS because you're using a nulled version of WP Rocket. You need an active license to use the Remove Unused CSS feature and further improve your website's performance..", 'rocket' );
+		$cta_message  = __( 'Click here to get a WP Rocket single license at 10%% off!', 'rocket' );
```

Comment:
I would use a `sprintf()` here to remove the 10 from the translatable string, since we don't want it be modifiable